lib/render_async/view_helper.rb

```diff
@@ -4,6 +4,9 @@ module RenderAsync
   module ViewHelper
 
     def render_async(path)
+      cached_view = Rails.cache.read("views/#{path}_cached")
```

Comment:
I think params would be picked up in the `path` variable.

Great question by the way, there should be a test for this so we are 100% sure :)